src/apps/backend/modules/shared-task-request/rest-api/shared-task-request-router.ts

```diff
@@ -0,0 +1,15 @@
+import { accessAuthMiddleware } from '../../access-token';
+import { ApplicationRouter } from '../../application';
+import { SharedTaskRequestController } from './shared-task-request-controller';
+
+export default class SharedTaskRequestRouter extends ApplicationRouter {
+  configure(): void {
+    const { router } = this;
+    const controller = new SharedTaskRequestController();
+
+    router.use(accessAuthMiddleware);
+
+    router.post('/shared-task', controller.createSharedTask);
+    router.get('/shared-tasks', controller.getSharedTasks);
```

Comment:
This API is not required at all. We already have an API to get all tasks. Use that API only by adding a boolean-type query param  `sharedTask`.